### ISSUE TITLE
Docs are confusing - do we support svgedit-config-iife.js/svgedit-config-es.js configuration mechanisms?

### ISSUE THREAD

**Issue by bluetwelve** [SVG-Edit/svgedit#833]
Documentation states that "Note that one may create a svgedit-config-iife.js file within the project root directory as of v3.0 (or for v2.8, within editor/config.js) and add such configuration directives to it without needing to modify the repository editor code" See https://github.com/SVG-Edit/svgedit/blob/master/docs/tutorials/ConfigOptions.md

This does not appear to be supported? Or have I missed something obvious.

**Comment by jfhenon** [comment]
Indeed, this is not supported anymore. I agree the documentation needs to be updated.
Thanks

**Comment by bluetwelve** [comment]
Thanks, it is a very minor point and not a request for anything concrete, I just got confused. There are a number of comments in the code that reference these files as well.

**Comment by Salamandar** [comment]
Hi, this would be nice to clean up the documentation about this :D

Is there any other way of configuring the app server-side ? Thanks !
